aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -259,6 +257,8 @@ std::pair<const AnnotatedKernel&, const char*> OperatorEntry::computeDispatchTab
     //      fit 2.1 and we can remove 2.4 entirely.
     if (!has_backend_kernel && !catchAllKernel_.empty()) {
       TORCH_INTERNAL_ASSERT(catchAllKernel_.front().kernel.isValid());
+      // Prepare for catchAll removal, make sure it's not used in dispatchTable
```

Comment:
I'm not sure the assert above this one (line 260) makes sense anymore...

Reply:
Yea these two code blocks indeed can be removed, I want to keep them as it is so that if anything went wrong (there's other ways registering to catchAll that I'm not aware of), we can track them easily. 
Both code blocks are removed in the next PR ;)